common/build/eslint-config-fluid/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The following rules have been disabled in all configs because they conflict with
 
 -   [@typescript-eslint/brace-style](https://typescript-eslint.io/rules/brace-style)
 -   [unicorn/number-literal-case](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/number-literal-case.md)
+-   The following rules have been disabled for test code:
+
+-   [unicorn/prefer-module](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/prefer-module.md)
```

Comment:
```suggestion

The following rules have been disabled for test code:

-   [unicorn/prefer-module](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/prefer-module.md)
```

Reply:
Sorry, had autocomplete set for this. I'll do a follow-up PR to fix the formatting.